### PR TITLE
feat: developer portal projects landing page

### PR DESCRIPTION
**Pull request by thesanjeevsharma**
Part of #2001 

This PR adds the projects home page under the developer portal. Page is accessible on `/developer/projects`.
Once, we are done with the refactoring we can delete the `Projects` container and rename `DeveloperPortalProjects` to `Projects`.

### CODE REVIEW

**Comment by jamessspanggg** [comment]
take note that we need to change all the urls from `/projects` to `/developer/projects` in the future when we are done

File: src/containers/DeveloperPortalProjects/components/Breadcrumb/Breadcrumb.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    @media (max-width: 768px) {
+      margin: 12px 0;
+    }
+  }
+
+  &__link {
+    @extend %h4;
+    color: var(--color-gray-400);
+    font-weight: var(--font-weight-regular);
+    line-height: 14px; // allow text to be centered by setting line-height to be the same as font size
```
Comment (jamessspanggg) [inline src/containers/DeveloperPortalProjects/components/Breadcrumb/Breadcrumb.scss:34]: can just put `line-height: 1` haha

**Review by jamessspanggg — COMMENTED** [review]
Can fix the buttons at a smaller screen, i.e. mobile?

Screen Shot 2021-08-04 at 8 47 14 PM

File: src/containers/DeveloperPortalProjects/components/Breadcrumb/Breadcrumb.scss (hunk 1 above)
Comment (thesanjeevsharma) [reply src/containers/DeveloperPortalProjects/components/Breadcrumb/Breadcrumb.scss:34]: I did not make any changes to Breadcrumb and SideMenu component. These can be done later, separately.

**Review by jamessspanggg — APPROVED** [review]

**Pull request merged**